### PR TITLE
Unique

### PR DESCRIPTION
**Pull request by HalelyukaM**
Component abour product uniqueness created.
This one will also be used by Rostyslav.
Styles will be modified when Diana complete merging her branch with mixins.
Please check also the structure and naming of files and props, maybe you will have better ideas on this.

### CODE REVIEW

File: client/src/components/unique/Unique.jsx
Comment (illia-dobro) [inline client/src/components/unique/Unique.jsx:1]: With SCSS modules, I think we can improve how we write classes. So maybe like this (destructuring).

```
import { btn } from './Button.module.scss'
className={btn}
```

I will try it this way, but you can leave and not rewrite.

Is your eslinter not working? In React, the class attribute must be replaced with className.

File: client/src/components/unique/Unique.module.scss
Comment (illia-dobro) [inline client/src/components/unique/Unique.module.scss:1]: It is cool! This SCSS is looking nice and readable.

File: client/src/components/unique/Unique.jsx
Comment (illia-dobro) [reply client/src/components/unique/Unique.jsx:1]: Actually, we have ESLint that checks props validation, so we have to use prop types or disable it in ESLint.
It is about a children notification, for example: 'name' is missing in props validation.

File: client/src/components/unique/Unique.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const Unique = ({image={}, name='', position='', signature={}}) => {
+    return (
+        <div class={styles.unique}>
+            <img src={image.src} alt={image.alt} />
+            <div class={styles.unique__description}>
+                <img src={uniqueBrand} alt="brand Name"></img>
+                <h3 class={styles.unique__focus}>Each Boltaeva brand product is truly unique - everything is thought out to the smallest detail.</h3>
+                <p class={styles.unique__content}>The brand seeks to build respect among the audience for its products, so that the presence of the company's products is a sign of prestige, since the focus of the business is focused on exclusive sales, but not on the mass market.</p>
+                <div class={styles.unique__signature}>
+                    <div class={styles.unique__signature_person}>
```
[the remark this answers is not in the document]
Comment (HalelyukaM) [reply client/src/components/unique/Unique.jsx:14]: Thanks, deleted it

File: client/src/components/unique/Unique.jsx
Comment (HalelyukaM) [reply client/src/components/unique/Unique.jsx:1]: Other variants for this component were "productUniqueness", "QualityAccent" or "UniquenessAccent"... I just left the first word I saw the accent on in the text
I will also appreciate your ideas)

Comment (HalelyukaM) [reply client/src/components/unique/Unique.jsx:1]: updated

**Comment by HalelyukaM** [comment]
I also added flex-direction as props, as this component is used in few pages with different sequence of text and picture

File: client/src/components/unique/Unique.module.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
+.unique {
+    max-width: 1440px;
+    margin: 0 auto;
+    display: flex;
+    align-items: center;
+    color: #555555;
+
+    &__description {
```
Comment (Diana-Chassot) [inline client/src/components/unique/Unique.module.scss:8]: сделалю как у тебя c & , очень красиво =)

File: client/src/components/unique/Unique.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import PropTypes from 'prop-types';
+import styles from './Unique.module.css';
+import uniqueBrand from '../../assets/unique_brandName.png';
+
+const Unique = ({image = {}, representName = "", representPosition = "", representSignature = {}, direction = "row"}) => {
+    return (
+        <div className={styles.unique} style={{flexDirection: direction}}>
+            <img src={image.src} alt={image.alt} />
+            <div className={styles.unique__description}>
+                <img src={uniqueBrand} alt="Brand Name"></img>
```
Comment (Diana-Chassot) [inline client/src/components/unique/Unique.jsx:11]: logo можно взять с готового компонента Logo . Есть различая в цветах , которые можно задать. А так все отлично

**Review by Diana-Chassot — APPROVED** [review]

**Pull request merged**